### ISSUE TITLE
All Known Related Works problems

### ISSUE THREAD

**Issue by foxm** [blakearchive/archive#267]
1) lines aren't getting italicized when they're supposed to

2) something is amiss with Illustrations to Gray's Poems:

http://localhost:8002/blake/work/but335

The Bard, from GrayA Descriptive Catalogue of Pictures 
"Number IV.
," in Blake,
typographic exhibition catalogue, 1809. Bentley 32. Erdman pages 541-42.

the lines are out of order and not spaced correctly. i imagine this has something to do with the <i> tags in the relationship entry

3) and any time there's a link, a newline comes after it when it shouldn't, as in The Marriage of Heaven and Hell: http://localhost:8002/blake/work/mhh

notice:

Copy H, c. 1790 
, repainted c. 1821
Fitzwilliam Museum
Cambridge, England

it should read:

Copy H, c. 1790, repainted c. 1821
Fitzwilliam Museum
Cambridge, England

**Comment by queryluke** [comment]
@nathan-rice @foxm 

Right now the issue is that anything within an <i> tag is getting parsed out as the "title". this happens in lines 94-96 in `import.py`

So in the case of BUT335.info.xml
```
<relationship>
  "Number IV. <i>The Bard, from Gray</i>," in Blake, <i>A Descriptive Catalogue of Pictures</i><br/>
  typographic exhibition catalogue, 1809.  Bentley 32.  Erdman pages 541-42.
</relationship>
```

The resulting title becomes a concatenation of the `<i>` tags: **The Bard, from GrayA Descriptive Catalogue of Pictures**

First the bads need to be edited. We should stick with markdown. single `*` instead of <i>

**Comment by foxm** [comment]
@queryluke there's also the issue of line breaks happening when they shouldn't:

Copy H, c. 1790
, repainted c. 1821
Fitzwilliam Museum
Cambridge, England

the ',repainted c. 1821' should go up.

In mhh.info.xml it looks like this:

```
<relationship><link ptr="mhh.h" type="copy">Copy H, c. 1790</link>, repainted c.

            1821<br/>

            Fitzwilliam Museum<br/>

            Cambridge, England 

            

        </relationship>
```

**Comment by queryluke** [comment]
Concerning the extra line break, there isn't much I can do. That `<link>` tag gets parsed separately from the rest of the information. We have 2 options:
1. Don't render any line breaks. Which would result in a single paragraph or related work info. 
2. Put `, repainted c. 1821` in the `<link>` tag. This would keep it on the same line.

**Comment by queryluke** [comment]
Concerning markdown, I've added the parser but there are still some issues you may come across. Here is a screenshot of what mhh looked like [when I added the markdown parser](https://drive.google.com/file/d/0BxLeDaHVwlDzOUVBTzNMdEVXQ00/view?usp=sharing)

Note the 2 oddities **Copy H** and **Death's Door**. These are cause by unnecessary line breaks in the xml. See my [commit to the data repo](https://github.com/blakearchive/data/commit/e5c5d7fae10c3da73ec220b90aaaf047cd51b5e7)

If you find other works with the same problem or any weird rendering, let me know. But you should be able to fix it by removing the extra line breaks in the xml.

**Comment by foxm** [comment]
Ok, thanks. I'm seeing a couple other things:

1) some lines have italics, no italics, then italics. so *blah blah* aldkjf *aldjijeo*, which produces this sort of thing:

*Studies for Plate 11 of *The Grave, "Death's Door"
Pencil sketches, c. 1805. Butlin 632 recto and verso.
Museum of Art, Carnegie Institute
Pittsburgh, Pennsylvania
Related to Plate 17.

2) and in http://london.lib.unc.edu/blake/work/letters we're missing some info (Blake's Illustrations of Dante and Illustrations to Dante's "Divine Comedy"). the entries are there, but their info isn't. in the info file they look like:

```
<relationship>*Blake's Illustrations of Dante*
            <br/>
            <link ptr="bb448.1.spb.04" type="object">Object 4, &quot;The Baffled Devils Fighting&quot;</link>
            <br/>
            intaglio engraving, c. 1892
            <br/>
            Collection of Robert N. Essick 
            <br/>
            Altadena, California
            <br/>
            <br/>
            in letter of: 
            <br/>
            25 April 1827 
            <link ptr="lt25april1827.1.ltr.02" type="object">(Object 2, lines 10-11)</link></relationship>
```

**Comment by foxm** [comment]
also, suddenly i'm getting a whole bunch of errors importing the info files:

UG [39b01bfe] 	ERROR:import:bb456.info.xml
DEBUG [39b01bfe] 	ERROR:import:bb435.info.xml
DEBUG [39b01bfe] 	ERROR:import:s-los.info.xml
DEBUG [39b01bfe] 	ERROR:import:mhh.info.xml
DEBUG [39b01bfe] 	ERROR:import:but557.info.xml
DEBUG [39b01bfe] 	ERROR:import:america.info.xml
DEBUG [39b01bfe] 	ERROR:import:bb499.info.xml
ERROR:import:gravepd.info.xml
DEBUG [39b01bfe] 	ERROR:import:but550.info.xml
DEBUG [39b01bfe] 	ERROR:import:esxxi.info.xml
ERROR:import:nnr.info.xml
DEBUG [39b01bfe] 	ERROR:import:bb421.info.xml
DEBUG [39b01bfe] 	ERROR:import:s-exp.info.xml
DEBUG [39b01bfe] 	ERROR:import:milton.info.xml
ERROR:import:but551.info.xml
DEBUG [39b01bfe] 	ERROR:import:bb469.info.xml
DEBUG [39b01bfe] 	ERROR:import:homer.info.xml
DEBUG [39b01bfe] 	ERROR:import:bb466.info.xml
DEBUG [39b01bfe] 	ERROR:import:aro.info.xml
ERROR:import:bbwba14.info.xml
DEBUG [39b01bfe] 	ERROR:import:bb515.info.xml
DEBUG [39b01bfe] 	ERROR:import:abel.info.xml
ERROR:import:bb514.info.xml
DEBUG [39b01bfe] 	ERROR:import:jerusalem.info.xml
DEBUG [39b01bfe] 	ERROR:import:bb125.info.xml
DEBUG [39b01bfe] 	ERROR:import:but536.info.xml
DEBUG [39b01bfe] 	ERROR:import:biblicalwc.info.xml
DEBUG [39b01bfe] 	ERROR:import:laocoon.info.xml
DEBUG [39b01bfe] 	ERROR:import:bb471.info.xml
DEBUG [39b01bfe] 	ERROR:import:vda.info.xml
DEBUG [39b01bfe] 	ERROR:import:pid.info.xml
DEBUG [39b01bfe] 	ERROR:import:thel.info.xml
DEBUG [39b01bfe] 	ERROR:import:but538.info.xml
DEBUG [39b01bfe] 	ERROR:import:letters.info.xml
DEBUG [39b01bfe] 	ERROR:import:esxiii.info.xml
DEBUG [39b01bfe] 	ERROR:import:bb468.info.xml
DEBUG [39b01bfe] 	ERROR:import:but527.info.xml
DEBUG [39b01bfe] 	ERROR:import:bb470.info.xml
ERROR:import:but335.info.xml
DEBUG [39b01bfe] 	ERROR:import:but543.info.xml
DEBUG [39b01bfe] 	ERROR:import:cpd.info.xml
DEBUG [39b01bfe] 	ERROR:import:s-inn.info.xml
DEBUG [39b01bfe] 	ERROR:import:gravewc.info.xml
DEBUG [39b01bfe] 	ERROR:import:but828.info.xml
ERROR:import:but769.info.xml
DEBUG [39b01bfe] 	ERROR:import:bb126.info.xml
DEBUG [39b01bfe] 	ERROR:import:songsie.info.xml
DEBUG [39b01bfe] 	ERROR:import:gates-sexes.info.xml
DEBUG [39b01bfe] 	ERROR:import:bb467.info.xml
DEBUG [39b01bfe] 	ERROR:import:bb85.info.xml
DEBUG [39b01bfe] 	ERROR:import:ahania.info.xml
DEBUG [39b01bfe] 	ERROR:import:but812.info.xml
DEBUG [39b01bfe] 	ERROR:import:but528.info.xml
ERROR:import:bb136.info.xml
DEBUG [39b01bfe] 	ERROR:import:biblicaltemperas.info.xml
DEBUG [39b01bfe] 	ERROR:import:allegropenseroso.info.xml
ERROR:import:__contents__.xml
DEBUG [39b01bfe] 	ERROR:import:bb448.info.xml
DEBUG [39b01bfe] 	ERROR:import:bb74.info.xml
ERROR:import:but542.info.xml
DEBUG [39b01bfe] 	ERROR:import:but537.info.xml
DEBUG [39b01bfe] 	ERROR:import:urizen.info.xml
DEBUG [39b01bfe] 	ERROR:import:but244.info.xml
ERROR:import:b-los.info.xml
DEBUG [39b01bfe] 	ERROR:import:but544.info.xml
ERROR:import:gravewd.info.xml
DEBUG [39b01bfe] 	ERROR:import:europe.info.xml
DEBUG [39b01bfe] 	ERROR:import:but529.info.xml
DEBUG [39b01bfe] 	ERROR:import:bb128.info.xml
DEBUG [39b01bfe] 	ERROR:import:gates-child.info.xml
DEBUG [39b01bfe] 	ERROR:import:bb465.info.xml
DEBUG [39b01bfe] 	ERROR:import:butwba10.info.xml
DEBUG [39b01bfe] 	ERROR:import:bb504.info.xml

**Comment by queryluke** [comment]
@foxm I think I figured out a better way to clean up the markdown. I'm trimming addition whitespace on each line which seems to be fixing the odd rendering. So you shouldn't need to edit any more bads. Not 100%, but I think the trim with do the trick

**Comment by foxm** [comment]
@queryluke sounds good, luke, thanks. i'm trying to figure out what we can do with the multi-italics in one line: *dakjd* adkjdf *dlkjf*

**Comment by queryluke** [comment]
Ok, I've pushed the change and that seems to have fixed all the wonky displays. So you shouldn't need to edit any of the extra lines in the bads. Sorry, sometimes more elegant solutions don't come to me until later.

RE 1: The `*` need to touch the word. I'm not sure what is supposed to be italicized. 
If you want:
*Studies for Plate 11 of* The Grave, *"Death's Door"*
Your xml should be:
`*Studies for Plate 11 of* The Grave, *"Death's Door"*`

RE 2: fixed it with #390 

RE 3: There is another ticket, which I've commented on, I'll try to do some additional digging, but I'm a bit stumped

RE 4: I commented out the logger that was throwing these errors. Not sure why it was in import.py it wasn't responding to an actually error. It was just spitting out content, like `print` or `echo`
